### PR TITLE
Add prettier and isort to pre-commit.

### PR DESCRIPTION
**Pull request by Qalthos**
Depends-on: https://github.com/ansible/ansible-zuul-jobs/pull/1742

##### SUMMARY

This brings it in line with other collections.

##### ISSUE TYPE

- Misc Pull Request

### CODE REVIEW

**Comment by Qalthos** [comment]
recheck

**Comment by Qalthos** [comment]
recheck

**Comment by Qalthos** [comment]
recheck

**Comment by Qalthos** [comment]
recheck

**Comment by Qalthos** [comment]
recheck

**Comment by Qalthos** [comment]
recheck

**Comment by Qalthos** [comment]
recheck

**Comment by Qalthos** [comment]
recheck

**Comment by Qalthos** [comment]
recheck

**Comment by Qalthos** [comment]
recheck

File: .isort.cfg [hunk 1]
```diff
@@ -0,0 +1,4 @@
+[settings]
+known_first_party=ansible_collections.vyos.vyos
+line_length=79
```
Comment (KB-perByte) [inline .isort.cfg:3]: AFAIK this should be 100!

**Pull request merged**